demo/x11_rawfb/nuklear_rawfb.h

```diff
@@ -171,10 +171,15 @@ nk_image_blendpixel(const struct rawfb_image *img,
     const int x0, const int y0, struct nk_color col)
 {
     struct nk_color col2;
+    struct nk_color t;
     unsigned char inv_a;
     if (col.a == 0)
         return;
 
+    t = col;
```

Comment:
This is really too hacky and I can't accept it, sorry.

Reply:
No problem at all. Should now be properly fixed.